src/UKHO.ERPFacade.API/XmlTransformers/S100DataContentPublishedEventXmlTransformer.cs

```diff
@@ -58,6 +58,11 @@ private void BuildProductActions(S100EventData eventData, XmlDocument soapXml, X
         {
             foreach (var product in eventData.Products)
             {
+                if (product.Status.StatusName.ToLower() == "cancellation")
+                {
+                    product.Status.StatusName = product.Status.StatusName.ToLower();
+                }
+
```

Comment:
Use constant for better maintainability and avoiding hardcoding "cancellation". 
Additionally, apply StringComparison.InvariantCultureIgnoreCase for case-insensitive string comparison and then assign the lowercase value.
```suggestion

private const string CANCELLATION_STATUS = "cancellation";

if (string.Equals(product.Status.StatusName, CANCELLATION_STATUS, StringComparison.InvariantCultureIgnoreCase))
{
    product.Status.StatusName = CANCELLATION_STATUS;
}

```

Reply:
Done.